tests/test_dispatcher/test_deprecated.py

```diff
@@ -0,0 +1,46 @@
+import pytest
+
+from aiogram.dispatcher.event.observer import TelegramEventObserver
+from aiogram.dispatcher.router import Router
+
+OBSERVERS = {
+    "callback_query",
+    "channel_post",
+    "chosen_inline_result",
+    "edited_channel_post",
+    "edited_message",
+    "errors",
+    "inline_query",
+    "message",
+    "poll",
+    "poll_answer",
+    "pre_checkout_query",
+    "shipping_query",
+    "update",
+}
+
+
+def test_deprecated_handlers_name():
+    from aiogram import __version__
+
+    major, minor = map(int, __version__.split(".")[:-1])
+
+    if minor >= 2 and major >= 3:  # version >=3.2.*
+        do_assert = pytest.raises(AttributeError)
+    else:  # for versions <=3.2.* (we don't care if major is lesser than `3`)
+        do_assert = pytest.warns(DeprecationWarning)
+
+    router = Router()
+
+    with do_assert:
+        for decor in OBSERVERS:
+            getattr(router, decor + "_handler")
+
+        assert all(
```

Comment:
When you make this test parametrized you can use something like this:

```python3
with check_deprecated('3.2'):
    observer = getattr(router, observer_name)
    assert isinstance(observer, TelegramEventObserver)
```

And i think other validations is no more needed